docs/docs/migrating-from-v1-to-v2.md

```diff
@@ -316,6 +316,9 @@ The `sizes` and `resolutions` queries are deprecated in v2. These queries have b
 Update image query and fragment names:
 
 ```diff
+import React from "react"
++ import { graphql } from "gatsby"
```

Comment:
I think this line should just be there but not marked as addition - importing graphql was covered in one of earlier points

Reply:
Yeah, I was hesitant to add those. I will remove them!